### PR TITLE
Settings: Optimize publisher logos image sizes

### LINKED ISSUE #4816: Settings: Optimize publisher logos image sizes
## Feature Description

On the editor settings screen we currently display the publisher logos in their full size, which can be quite large.

Some thoughts:

* We should use `srcset` here for optimization.  
  (`calculateSrcSet` util might serve as inspiration)
* The publisher logos are displayed in 100x100px (`Logo` component). The AMP format uses/recommends 96x96 - let's also use that size? Otherwise a 96x96 image will get resized a bit.
* Let's set explicit `width` and `height` attributes on the `img`
* Pick reasonable sizes limit so we don't unnecessarily load large images.

## Additional Context

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

### PR DESCRIPTION
**Pull request by maxyinger**
## Context
Looked into utilizing srcsets, but there's no smaller sizes for logo resources available beyond what the user uploads

## Summary
Just sets the publisher logo size to 96x96 px in the settings view to make it more consistent with AMP recommendations.

## Relevant Technical Choices
NA

## To-do
NA

## User-facing changes
Logos should be slightly larger in the settings page.

## Testing Instructions
- Go to settings
- See that the Publisher logo thumbnails are 96x96 px

### QA

- [ ] This is a non-user-facing change and requires no QA

This PR can be tested by following these steps:

1.

### UAT
Follow testing instructions above

- [x] UAT should use the same steps as above.

This PR can be tested by following these steps:

1.

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [ ] This PR contains automated tests (unit, integration, and/or e2e) to verify the code works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #4816

### CODE REVIEW

**Comment by swissspidy** [comment]
From the ticket:

> Let's set explicit width and height attributes on the img

I think we can still do that. Helps prevent layout shifts.

**Comment by spacedmonkey** [comment]
Related https://github.com/google/web-stories-wp/issues/7337 https://github.com/google/web-stories-wp/issues/4148

**Review by BrittanyIRL — APPROVED** [review]
👍

**Review by swissspidy — CHANGES_REQUESTED** [review]
Let's add `width` and `height` to the images

**Comment by maxyinger** [comment]
realized there's a 2px border around the images so the columns/rows need to be 100x100 to accommodate for that and allow the image to be 96x96

File: assets/src/dashboard/app/views/editorSettings/components.js [hunk 1]
```diff
@@ -150,7 +150,10 @@ export const GridItemButton = styled.button`
   ${themeHelpers.focusableOutlineCSS};
 `;
 
-export const Logo = styled.img`
+export const Logo = styled.img.attrs({
+  height: '96px',
+  width: '96px',
```
Comment (swissspidy) [inline assets/src/dashboard/app/views/editorSettings/components.js]:
```suggestion
  height: 96,
  width: 96,
```


These attributes are unit-less :-)

**Comment by swissspidy** [comment]
> realized there's a 2px border around the images so the columns/rows need to be 100x100 to accommodate for that and allow the image to be 96x96

There's also a 1px border on `GridItemContainer`, so the image is now actually displayed at 94x94px

Screenshot 2021-07-14 at 10 52 21

File: assets/src/dashboard/app/views/editorSettings/components.js [hunk 2]
```diff
@@ -109,8 +109,8 @@ export const Error = styled(CenterMutedText)`
 
 export const UploadedContainer = styled.div`
   display: grid;
-  grid-template-columns: repeat(auto-fill, 56px);
-  grid-auto-rows: 56px;
+  grid-template-columns: repeat(auto-fill, 100px);
+  grid-auto-rows: 100px;
```
Comment (swissspidy) [inline assets/src/dashboard/app/views/editorSettings/components.js]:
```suggestion
  grid-template-columns: repeat(auto-fill, 102px);
  grid-auto-rows: 102px;
```

To accommodate for the other border as well so images are actually displayed at 96x96

**Review by swissspidy — APPROVED** [review]

**Pull request merged**